pandas/io/gbq.py

```diff
@@ -148,122 +341,201 @@ def run_query(self, query):
             'configuration': {
                 'query': {
                     'query': query
-                    #'allowLargeResults', 'createDisposition', 'preserveNulls', destinationTable, useQueryCache
+                    # 'allowLargeResults', 'createDisposition', 'preserveNulls', destinationTable, useQueryCache
                 }
             }
         }
 
         try:
-            query_reply = job_collection.insert(projectId=self.project_id,
-                                           body=job_data).execute()
-            status = query_reply['status']
+            query_reply = job_collection.insert(projectId=self.project_id, body=job_data).execute()
         except AccessTokenRefreshError:
```

Comment:
you can certainly *catch* exceptions raised within gbq, but usually best just to let them bubble up, unless you can actually handle them (e.g. if the credentials are screwed up, then nothing you can do, so don't even catch it in the first place). 

All that said if their is an exception (or family) of exceptions that could use a more intuitve error message then pls do so.